### ISSUE TITLE
Alternative  workflow in "Give feedback" page

### ISSUE THREAD

**Issue by fgailly** [dodona-edu/dodona#2443]
The "Give feedback" page now only supports the following workflow:
- evaluate submission exercise 1
- go to next submission of random student for exercise 1

When using the evaluation mode in combination with a rubric, the following workflow should be supported:
- evaluate submission student A exercise 1
- evaluate submission student A exercise 2
 ...
- evaluate submission student B exercise 1
- evaluate submission student B exercise 2

It should be possible in the "give feedback" page  to evaluate submissions like they are presented in the table in the "Evaluation" page = row by row

**Comment by pdawyndt** [comment]
At the bottom of the panel in the right margin, you can navigate to the other assignments for the current student. The number of assignments is usually much smaller than the number of students, so that's why the UI provides different ways to navigate to other student (same exercise) and other exercise (same student).

![image](https://user-images.githubusercontent.com/5736113/103518376-f4312880-4e73-11eb-84b7-1b789ca02f59.png)

**Comment by fgailly** [comment]
yes navigation to other assignments is possible but before I can do this, I need to press "mark as done". Additionally in order to go to the next student, I have to go to the evaluate page and their select the first assignment of the next student in the alphabet. I am currently evaluating an exam with three assignments and 240 students which means for every student three additional clicks  and two clicks to go to the next student + scrolling  down for selecting the next student.

**Comment by pdawyndt** [comment]
Would you think that assessing per student is more efficient than assessing per exercise (I would think the opposite)?

Apart from efficiency, it is [also recommended](https://onderwijstips.ugent.be/nl/tips/verbeteren-hoe-doe-je-dat-correct-en-efficient/) to evaluate per exercise and not per student in order to avoid some biases during assessment. That's also why Dodona navigates between students in a random order, and the order is different per exercise. This is a design choice, in addition with pseudonimization during assessment (which will be useful in full strength once we also give the option to add scores in Dodona itself; expected for the release after the exam period).

**Comment by fgailly** [comment]
I agree, but now I am using the dodona evaluate feature in combination with a rubric on ufora. This rubric is defined for the grading the complete exam + ufora also navigates alphabetically between students. Of course the option to add scores in dodona could solve my problem. However I still believe that lecturers should be able to decide how they want to evaluate : assignment by assignment or student by student.
